src/components/Menu/Menu.tsx

```diff
@@ -228,6 +228,7 @@ export const Menu: React.FC<IMenuProps> = (
                 )}
                 onMouseEnter={onMouseEnter}
                 onMouseLeave={onMouseLeave}
+                data-testid="moonstone-menu"
```

Comment:
Same here, can we add this attribute for test only?

Reply:
Yes!